doc/tutorial/extending_theano.txt

```diff
@@ -259,7 +259,7 @@ Op Example
 
         def make_node(self, x):
             # check that the theano version has support for __props__
-            assert hasattr(self, '_props')
+            assert hasattr(self, '__props__')
```

Comment:
My vote is to remove the check completely since \__props__ is old enough that not many people should have a theano without it and it was in the latest release.

But if we are to keep the check, then it should go back to _props, of course.